airflow/data/agencies.yml

```diff
@@ -294,6 +302,14 @@ commerce-municipal-bus-lines:
       gtfs_rt_service_alerts_url: https://citycommbus.com/gtfs-rt/alerts
       gtfs_rt_trip_updates_url: https://citycommbus.com/gtfs-rt/tripupdates
   itp_id: 75
+commute-org-shuttles:
```

Comment:
This is a precursor feed to the bay area 511 feed, so duplicative of datasets.

Reply:
This one looks to be an active Trillium feed, and it's not completely identical to what I downloaded for commute shuttles from 511. Either way, shouldn't they be in the .yml file so we can easily get validation results for them and make sure they end up on reports.calitp? Right now, I have them marked as missing from the reports site. We have close to all the other bay area services in the .yml file/reports site even though they're technically part of the 511 feed too. 

I guess what I'm trying to say, for the purpose of clarity, is why wouldn't we include this feed, when we have other bay area operator feeds in our database outside of 511? (I.e. Westcat has a Trillium static link, Samtrans has their own static link, etc.)

Reply:
> This one looks to be an active Trillium feed, and it's not completely identical to what I downloaded for commute shuttles from 511

Reply:
There isn't much cost to ingest this information and it is a somewhat separate issue from which feed the assessment process selects. I say we might as well be including these feeds and if there is ever a case that an agency goes rogue and decides to not be aggregated, then we'll have the history.